src/app/store/quest.ts

```diff
@@ -28,9 +59,65 @@ export function createQuestStore({
 	const dailyQuests = computed(() =>
 		_allQuests.value.filter(i => i.repeat_type === QuestRepeatType.daily)
 	);
-	const quests = computed(() =>
-		_allQuests.value.filter(i => i.repeat_type !== QuestRepeatType.daily)
-	);
+
+	const questChunks = computed(() => {
```

Comment:
i want to be conscious of the amount of code we have to load in for the shell. let's see if we can maybe move this into other places to only load on demand